### ISSUE TITLE
DROP SHARD doesn't remove data from _internal

### ISSUE THREAD

**Issue by beckettsean** [influxdata/influxdb#6988]
### Bug report

__System info:__ 
InfluxDB 0.13

__Steps to reproduce:__

```
> drop shard 257
> SELECT "writePointsOk",id  FROM "_internal".."shard" WHERE time > now() - 10s group BY *
name: shard
tags: clusterID=36934674580451795, database=_internal, engine=tsm1, hostname=sean-stable, id=252, nodeID=0, path=/var/lib/influxdb/data/_internal/monitor/252, retentionPolicy=monitor
time			writePointsOk	id
----			-------------	--
1468005095000000000	317511		252

<snip>

name: shard
tags: clusterID=36934674580451795, database=mydb, engine=tsm1, hostname=sean-stable, id=257, nodeID=0, path=/var/lib/influxdb/data/mydb/default/257, retentionPolicy=default
time			writePointsOk	id
----			-------------	--
1468005095000000000	1		257

<snip>
> 
```

(The shard only had one point before being dropped, so that's consistent output)

__Expected behavior:__

Once dropped, the shard should not appear in `_internal` monitoring. For shards removed by retention policy expiration things work as expected, but for manually dropped shards the data persists.

__Actual behavior:__ 

Statistics about shards that have been dropped by `DROP SHARD` persist in `_internal`

__Additional info:__

**Comment by beckettsean** [comment]
@rkuchan, nice catch

@jsternberg is in the midst of a PR that should resolve this.

**Comment by jsternberg** [comment]
This should be fixed with #6964. If this is still an issue with that fix, please reopen so I can take another look and address the problem.
